test/uk/gov/hmrc/pillar2/controllers/ObligationControllerSpec.scala

```diff
@@ -0,0 +1,64 @@
+package uk.gov.hmrc.pillar2.controllers
+
+import org.mockito.Mockito.when
+import org.scalatestplus.scalacheck.ScalaCheckPropertyChecks
+import play.api.inject.bind
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.libs.json.Json
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import play.api.{Application, Configuration}
+import uk.gov.hmrc.auth.core.AuthConnector
+import uk.gov.hmrc.pillar2.connectors.ObligationConnector
+import uk.gov.hmrc.pillar2.controllers.auth.{AuthAction, FakeAuthAction}
+import uk.gov.hmrc.pillar2.generators.Generators
+import uk.gov.hmrc.pillar2.helpers.BaseSpec
+import uk.gov.hmrc.pillar2.models.obligation.ObligationStatus.Fulfilled
+import uk.gov.hmrc.pillar2.models.obligation.{ObligationInformation, ObligationType}
+
+import java.time.LocalDate
+import scala.concurrent.Future
+
+class ObligationControllerSpec extends BaseSpec with Generators with ScalaCheckPropertyChecks {
+
+  val PlrReference       = "XMPLR0123456789"
```

Comment:
I think it's worth making this camel case otherwise in an ide it looks like a class or something:
```
plrReference
```

Reply:
Ah so we're treating it as a constant